Research/QuantBook.cs

```diff
@@ -656,5 +766,119 @@ private object GetPropertyValue(object baseData, string fullName)
 
             return baseData;
         }
+
+        /// <summary>
+        /// Convert Python input to a list of Symbols
+        /// </summary>
+        /// <param name="input">Object with the desired property</param>
+        /// <returns>List of Symbols</returns>
+        private IEnumerable<Symbol> PyConvertToSymbols(PyObject input)
```

Comment:
Nice method! Thinking we might wanna move it into the Common extensions/utils section so it can be reused and we can add some simple tests for it covering his behavior